docs/sources/developers/http_api/_index.md

```diff
@@ -29,13 +29,59 @@ Starting from version 9.1, there is also a [OpenAPI v3 specification](https://ed
 
 Users can browser and try out both via the Swagger UI editor (served by the grafana server) by navigating to `/swagger-ui` and `/openapi3` respectively.
 
+## Authenticating API requests
```

Comment:
This content has been moved from `auth.md` file and only slightly changed to reference service accounts not API keys.